src/app/components/client/exposure_card/ExposureCard.module.scss

```diff
@@ -48,6 +48,10 @@
     justify-content: space-between;
     height: 30px; // fixed height to standardize image/icon heights
 
+    .hideOnMobile {
+      display: none;
+    }
```

Comment:
Oh sorry, actually, I just did notice another problem: `.hideOnMobile` is not limited to mobile (i.e. the media query that shows it on desktop does not apply to the `dt`), and so it no longer gets announced when the value _is_ visible.